### ISSUE TITLE
💄 Improve Print Quality

### ISSUE THREAD

**Issue by SteffenKn** [process-engine/bpmn-studio#510]
#### Describe your issue

The print function still uses a png to print the diagram. This leads to a low print quality and should be improved.

#### How can others reproduce the issue?

- Print a diagram

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by robpal94** [comment]
I encountered the following lines: 

```
const pixelRatio: number = window.devicePixelRatio || 1;
canvas.width = svgWidth * pixelRatio;
canvas.height = svgHeight * pixelRatio;
```

If you search for documentation of the `window.devicePixelRatio` property you find the following: 

> The Window property devicePixelRatio returns the ratio of the resolution in physical pixels to the resolution in CSS pixels for the current display device. This value could also be interpreted as the ratio of pixel sizes: the size of one CSS pixel to the size of one physical pixel. In simpler terms, this tells the browser how many of the screen's actual pixels should be used to draw a single CSS pixel.

[Source](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio)

Does that mean, that the quality of my **print** depends on the **pixel ratio of my device**?

**Comment by SteffenKn** [comment]
This was used to improve the quality of the exported png, since the exported png looked terrible on retina screens.
Because the exporting and printing use the same converting this means that the print quality depends on the pixel ratio.
But since we do not know in which size the diagram should be printed, it does not help us to simply increase the pixel ratio.

**Comment by robpal94** [comment]
If we want a really **really** dirty _hotfix_, we can change the `pixelRatio` to a constant value of 4 because thats the ratio we need to create a canvas with a pixel density of 300 DPI (which is the standard pixel density for prints in the dina4 format). 

But still, the problem with bigger formats, such as A3, remains.
